### ISSUE TITLE
Getting some strange output

### ISSUE THREAD

**Issue by kolchy** [sitereactor/umbraco-console-example#2]
When just listing the content nodes, I get the following. Any ideas??
## Application Started

ApplicationContext is available: True
DatabaseContext is available: True
## Database is available: True

-- Options --
List content nodes: l
Create new content: c
Create Umbraco database schema in empty db: d
Quit application: q
l
STRING TYPE Alias
CASE UmbracoCase
CHAR 'H' term upper - 1 - 0/0 leading
CHAR 'o' term !upper - 2 - 1/0 !leading
CHAR 'm' term !upper - 3 - 2/0 !leading
CHAR 'e' term !upper - 3 - 3/0 !leading
CHAR 'P' term upper - 3 - 4/0 !leading
CHAR 'a' term !upper - 3 - 5/0 !leading
CHAR 'g' term !upper - 3 - 6/0 !leading
CHAR 'e' term !upper - 3 - 7/0 !leading
TERM "HomePage" word UmbracoCase
STRING TYPE Alias
CASE UmbracoCase
CHAR 'H' term upper - 1 - 0/0 leading
CHAR 'o' term !upper - 2 - 1/0 !leading
CHAR 'm' term !upper - 3 - 2/0 !leading
CHAR 'e' term !upper - 3 - 3/0 !leading
CHAR 'P' term upper - 3 - 4/0 !leading
CHAR 'a' term !upper - 3 - 5/0 !leading
CHAR 'g' term !upper - 3 - 6/0 !leading
CHAR 'e' term !upper - 3 - 7/0 !leading
TERM "HomePage" word UmbracoCase
STRING TYPE Alias
CASE UmbracoCase
CHAR 'M' term upper - 1 - 0/0 leading
CHAR 'a' term !upper - 2 - 1/0 !leading
CHAR 's' term !upper - 3 - 2/0 !leading
CHAR 't' term !upper - 3 - 3/0 !leading
CHAR 'e' term !upper - 3 - 4/0 !leading
CHAR 'r' term !upper - 3 - 5/0 !leading
CHAR 'L' term upper - 3 - 6/0 !leading
CHAR 'a' term !upper - 3 - 7/0 !leading
CHAR 'y' term !upper - 3 - 8/0 !leading
CHAR 'o' term !upper - 3 - 9/0 !leading
CHAR 'u' term !upper - 3 - 10/0 !leading
CHAR 't' term !upper - 3 - 11/0 !leading
TERM "MasterLayout" word UmbracoCase
STRING TYPE Alias
CASE UmbracoCase
CHAR 'M' term upper - 1 - 0/0 leading
CHAR 'a' term !upper - 2 - 1/0 !leading
CHAR 's' term !upper - 3 - 2/0 !leading
CHAR 't' term !upper - 3 - 3/0 !leading
CHAR 'e' term !upper - 3 - 4/0 !leading
CHAR 'r' term !upper - 3 - 5/0 !leading
TERM "Master" word UmbracoCase

**Comment by sitereactor** [comment]
Yeah, I believe that is because of a rogue Console.WriteLine in the Core. Which version of umbraco did you try this with?

**Comment by kolchy** [comment]
7.1.4...
